android/PhysicalWeb/app/src/main/java/org/physical_web/demos/FatBeaconHelloWorld.java

```diff
@@ -27,7 +27,7 @@
  */
 public class FatBeaconHelloWorld implements Demo {
   private static final String TAG = FatBeaconHelloWorld.class.getSimpleName();
-  private static boolean mIsDemoStarted = false;
+  private boolean mIsDemoStarted = false;
```

Comment:
I think this needs to be static because it needs to retain that state even when the user goes back to the list.